assets/pages/digital-subscription-checkout/helpers/user.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import { type Option } from 'helpers/types/option';
+
+export type User = {|
+  firstName: Option<string>,
+  lastName: Option<string>,
+  email: Option<string>,
+|};
+
+function getUser(): User {
+
+  if (window && window.guardian && window.guardian.user) {
+    return {
+      firstName: window.guardian.user.firstName || null,
+      lastName: window.guardian.user.lastName || null,
+      email: window.guardian.user.email || null,
```

Comment:
You may need to do a `typeof a === 'string'` check here to prevent other types making it through.

Reply:
Good idea. I've tried to add this check.